common/options/options.go

```diff
@@ -439,6 +446,20 @@ func (opts *ToolOptions) AddToExtraOptionsRegistry(extraOpts ExtraOptions) {
 	opts.URI.extraOptionsRegistry = append(opts.URI.extraOptionsRegistry, extraOpts)
 }
 
+func (opts *ToolOptions) CallArgParser(args []string) ([]string, error) {
+	args, err := opts.parser.ParseArgs(args)
+	if err != nil {
+		return []string{}, err
+	}
+
+	// Set VerbosityParsed flag to make sure we reset verbosity level when we call ParseArgs again
+	if opts.Verbosity.VLevel != 0 && !opts.VerbosityParsed {
+		opts.VerbosityParsed = true
+	}
+
+	return args, err
```

Comment:
```suggestion
	return args, nil
```
[nit] Just to be explicit that we won't have an actual error here.